examples/image_classification/ImageClassificationExample.js

```diff
@@ -28,6 +30,7 @@ class ImageClassificationExample extends BaseCameraExample {
   /** @override */
   _processExtra = (output) => {
     let labelClasses;
+    console.log(output);
```

Comment:
@lionkunonly Please remove this `console.log` line, thanks.

Reply:
Done